pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -479,6 +480,8 @@ def _choose_accelerator(self) -> str:
                 return "ipu"
             if _HPU_AVAILABLE:
                 return "hpu"
+            if MPSAccelerator.is_available():
```

Comment:
Shouldn't this come after cuda?
On a M1 with an additional CUDA GPU, which one would we prefer? Probably the CUDA, right?

Reply:
that isn’t even a possible combination. Apple does not have support for cuda at all